samples/Samples/Pages/OAuthExample.cshtml

```diff
@@ -0,0 +1,81 @@
+﻿@page
+@model Samples.Pages.OAuthExampleModel
+@{
+}
+
+<h1>OAuth with ASP.NET Core and Amazon Cognito</h1>
+
+<p>
+    Cognito support OAuth 2.0 authorization framework to make authenticated REST API calls. To identify the authenticated user to an
+    API a JSON Web Token (JWT) is generated by the OAuth provider, in this case Cognito, and passed as the <b>Authorization</b> header
+    to the REST API. The format of the value for Authorization header is "<i>bearer &lt;jwt-token&gt;</i>".
+</p>
+
+<h2>Retrieving current user's JWT token.</h2>
+
+<p>
+    In an ASP.NET Core application a JWT token can be retrieved for the current logged in user by injecting ASP.NET Core's UserManager&lt;CognitoUser&gt; service.
+    Using the UserManager service the CognitoUser object can be retrieved and the ID token can be retrieved. The ID token is used as the JWT token that can be used to
+    make authenticated REST API calls.
+</p>
+<p>
+    Here is the snippet of code that shows how to retrieve the current logged in users JWT token.
+</p>
+<pre><code>
+public async Task OnGet()
+{
+    var cognitoUser = await _userManager.GetUserAsync(this.HttpContext.User);
+    IsLoggedIn = cognitoUser != null;
+    if (IsLoggedIn)
+    {
+        this.JwtToken = cognitoUser.SessionTokens.IdToken;
+    }
+}
+</code></pre>
+
+
+@if (Model.IsLoggedIn)
+{
+<p>This is the current logged in user's JWT token:</p>
+<textarea disabled style="width: 100%; box-sizing: border-box;" rows="8">@Model.JwtToken</textarea>
```

Comment:
This didn't work for me:
![image](https://user-images.githubusercontent.com/1170880/128396521-362c216f-f007-4a4d-9cc6-bfeefa8e72e5.png)

Running this branch from source, filled out `appsettings.Development.json`.

Reply:
It worked for me

![image](https://user-images.githubusercontent.com/6940508/201346084-ec43a875-8fb8-4426-9e37-a5ad37b41371.png)